react/interfaces.tsx

```diff
@@ -0,0 +1,15 @@
+import { VideoHTMLAttributes } from 'react'
+
+export interface VideoPlayer extends VideoHTMLAttributes<HTMLVideoElement> {
+  name?: string
+  description?: string
+  src: string
+  type?: string
+  controlsType?: string
+  IconPlay?: any
+  IconPause?: any
+  IconFullscreen?: any
+  IconVolumeOn?: any
+  IconVolumeOff?: any
```

Comment:
Same thing here. These `any`s could cause issues in the future, with people importing components and trying to pass any values to them